networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -12,6 +12,22 @@ The Network Observability Operator enables administrators to observe and analyze
 These release notes track the development of the Network Observability Operator in the {product-title}.
 
 For an overview of the Network Observability Operator, see xref:../../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
+[id="network-observability-operator-release-notes-1-4-1"]
+== Network Observability Operator 1.4.1
+The following advisory is available for the Network Observability Operator 1.4.1:
+
+* link:https://access.redhat.com/errata/RHSA-2023:122114[2023:122114 Network Observability Operator 1.4.1]
+
+=== CVEs
+* link:https://access.redhat.com/security/cve/cve-2023-44487[2023-44487]
+* link:https://access.redhat.com/security/cve/cve-2023-39325[2023-39325]
+
+== Bug fix
+
+* Previously, when sending to Kafka, the Kafka message key was ignored causing an error with connection tracking. Now the key is used for partitioning, so each flow from the same connection are sent to the same processor. (link:https://issues.redhat.com/browse/NETOBSERV-926[*NETOBSERV-926*])
```

Comment:
just wondering: should we say this was previously mentioned as a known issue, hence that known issue can be ignored now? Or we can consider this is implicit.

Reply:
I actually think the known issue should be left in 1.4 and then mention that it was previously a known issue in 1.4.1 bug fix note, like your first question was asking. I don't think its right to edit out the known issue.

Reply:
Just updated this!